docs: adds an installation guide (#479)
asyncapi/glee: docs/installation.md

---
title: 'Installation guide'
weight: 30
---

## Glee Installation

Before installing Glee framework into your project, make sure you have pre-installed AsyncAPI CLI tool under the hood.

The best way to get started with Glee is by using AsyncAPI CLI, which sets up everything automatically for you. 
To create a project, run:
```sh
asyncapi new glee
```

> <Remember>
> For more information on how to install AsynAPI CLI, you can review <a href="https://www.asyncapi.com/docs/tools/cli/installation">CLI installation documentation</a>.
> </Remember>

Reviewer: ```suggestion
> For more information on how to install the AsynAPI CLI, you can review the [CLI installation guide](https://www.asyncapi.com/docs/tools/cli/installation).
```
